articles/quickstart/native/android/_includes/_login.md

```diff
@@ -20,94 +20,48 @@ private Auth0 auth0;
 
 @Override
 protected void onCreate(@Nullable Bundle savedInstanceState) {
-    //...
+    // ...
     auth0 = new Auth0(this);
     auth0.setOIDCConformant(true);
 }
 ```
 
-If you prefer to hardcode the resources, use the constructor that receives both strings. 
-
-Finally, create a `login` method and use the `WebAuthProvider` class to authenticate with any connection you enabled on your application in the [Auth0 dashboard](${manage_url}/#/). 
+Otherwise, use the constructor that receives both strings.
 
-After you call the `WebAuthProvider#start` function, the browser launches and shows the **Lock** widget. Once the user authenticates, the callback URL is called. The callback URL contains the final result of the authentication process.
+Finally, create a `login` method and use the `WebAuthProvider` class to authenticate with any connection you enabled on your application in the [Auth0 dashboard](${manage_url}/#/).
 
 ```java
 // app/src/main/java/com/auth0/samples/LoginActivity.java
 
 private void login() {
     WebAuthProvider.login(auth0)
-        .withScheme("demo")
-        .withAudience(String.format("https://%s/userinfo", getString(R.string.com_auth0_domain)))
-        .start(MainActivity.this, new AuthCallback() {
-            @Override
-            public void onFailure(@NonNull Dialog dialog) {
-               // Show error Dialog to user
-            }
-
-            @Override
-            public void onFailure(AuthenticationException exception) {
-               // Show error to user
-            }
-
-            @Override
-            public void onSuccess(@NonNull Credentials credentials) {
-               // Store credentials
-               // Navigate to your main activity
-            }
-    });
-}
-```
-
-::: note
-Future steps of this quickstart guide move the `AuthCallback` in-line instance into a class field named `loginCallback`.
-:::
-
-### Capture the Result
+            .withScheme("demo")
+            .withAudience(String.format("https://%s/userinfo", getString(R.string.com_auth0_domain)))
+            .start(MainActivity.this, new AuthCallback() {
+                @Override
+                public void onFailure(@NonNull Dialog dialog) {
+                // Show error Dialog to user
```

Comment:
Indentation of the comments (here and below)

Reply:
Fixed in 8e7e072b345e65b094a662b6e0c95dcd502ebd7f.